src/sentry/monitors/consumers/monitor_consumer.py

```diff
@@ -811,20 +811,14 @@ def _process_checkin(item: CheckinItem, txn: Transaction | Span) -> None:
             # 03-B
             # Create a brand new check-in object
             except MonitorCheckIn.DoesNotExist:
-                # Infer the original start time of the check-in from the duration.
-                # Note that the clock of this worker may be off from what Relay is reporting.
-                date_added = start_time
-                if duration is not None:
-                    date_added -= timedelta(milliseconds=duration)
```

Comment:
Do we rely on this date at all for calculating lateness and so on?

Reply:
no. in fact, this is what makes it hard to diagnose why a miss is created when it "appears" that a check in was on time.

If you have a job that takes 15m, but your miss window is 10m, then we will create a miss for your job at 12:00, but if you send an OK at 12:15, it will be back dated to 12:00 and it will _appear_ that your job was on time and the miss was erroneous.